packages/app/components/strategies/StrategyCard.tsx

```diff
@@ -15,8 +19,8 @@ interface StrategyCardProps {
 }
 
 export const StrategyCard = ({ strategy }: StrategyCardProps) => {
-  const { selectedStrategy, setSelectedStrategy, setShouldResetStackbox } =
-    useStrategyContext();
+  const { resetDefaultFormValues } = useStackboxFormContext();
```

Comment:
I think we can name it `resetFormValues` , by reseting I assume it goes to default. 

```suggestion
  const { resetFormValues } = useStackboxFormContext();
```